sub-packages/bionemo-scdl/README.md

```diff
@@ -136,11 +136,11 @@ convert_h5ad_to_scdl --data-path hdf5s --save-path example_dataset
 
 The runtime and memory usage are examined on a CellXGene Dataset with ~1.5 million rows and a size of 24 GB. On this dataset, there is a 4.9x memory speed up.
 
-<img src="images/throughput.png" alt="Throughput Image" width="600">
+<img src="https://raw.githubusercontent.com/NVIDIA/bionemo-framework/main/sub-packages/bionemo-scdl/images/throughput.png" alt="Throughput Image" width="600">
```

Comment:
Is there something like a relative path that would work for all of the ways this could be displayed?
* Github nav render (prob works fine)
* Compiled docs (the github main docs page would be fine, since that points at main like this link)
* Versioned compiled docs (this would not work here since it points to the latest image).